.env.exampleV2.holesky

```diff
@@ -49,6 +51,12 @@ EIGENDA_PROXY_EIGENDA_V2_BLOB_PARAMS_VERSION=0
 # sent, decreasing this value is a crude sort of optimization.
 EIGENDA_PROXY_EIGENDA_V2_MAX_BLOB_LENGTH=1MiB
 
+# Address of the EigenDA service manager contract (required for validator retrieval)
+EIGENDA_PROXY_EIGENDA_V2_SERVICE_MANAGER_ADDR=0xD4A7E1Bd8015057293f0D0A557088c286942e84b
+
+# Address of the BLS operator state retriever contract (required for validator retrieval)
+EIGENDA_PROXY_EIGENDA_V2_BLS_OPERATOR_STATE_RETRIEVER_ADDR=0x003497Dd77E5B73C40e8aCbB562C8bb0410320E7
+
```

Comment:
is that not readable from the service manager?

Reply:
Yeah its not because stateRetriever was meant to be a standalone shared contract that everyone would use (all its calls take a serviceManagerAddr as argument). But in the end each team ends up deploying their own anyways so.. we're stuck with this.